README.md

```diff
@@ -432,3 +433,24 @@ If you have an idea for a new feature, don't hesitate to share it on the [issues
 <!-- prettier-ignore-end -->
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
+
+
+### Tips
+
+1. When running PySR with `julia 1.11`, the process seems to run into a memory leak bug that halts the execution by exceeding the available memory. Forcing the version `1.10` can help avoiding such problem:
+
+```python
+import juliapkg
+juliapkg.require_julia("~1.10")
+```
+
+2. Another memory issue can happen if using a large enough `maxsize` parameter **(Miles: is there any explanation for that?)**. Since the main usage of PySR is for the discovery of scientific equations, this value should be set small anyway. Anything beyond $50$ seems to create a significant slowdown and memory usage.
```

Comment:
> Anything beyond $50$ seems to create a significant slowdown and memory usage.

I think we can turn this off now. It was basically only there because some beginners were running with like 10,000 maxsize, so I wanted it to warn them 😄

Reply:
$10,000$ 😶 lol, I wonder why